### PR TITLE
MAUI-3668 UG for Pyramid Chart

### PR DESCRIPTION
**Pull request by SaiyathAliFathima**
### Feature description ###
Preparing UG document for pyramid and funnel charts.

### Analysis and design ###
Referred the Circulart chart UG - https://help.syncfusion.com/maui/circular-charts/getting-started?cs-save-lang=1&cs-lang=csharp , pyramid and funnel UG in WinUI and flutter platforms - https://help.syncfusion.com/winui/pyramid-chart/getting-started ,https://help.syncfusion.com/flutter/pyramid-chart/chart-appearance

### Solution description ###
This UG document helps to guide the user to understand the feature.

### Areas affected and ensured ###
None

### API Changes ###
None

### Behavioral Changes ###
None

### Test cases ###
Not Applicable

### Testbed sample location ###
Not Applicable

### Does it have any known issues?
None

### Does it have memory leak?
None

### CODE REVIEW

File: MAUI/Pyramid-Charts/Getting-Started.md [hunk 1]
```diff
@@ -109,17 +110,17 @@ Next, create a view model class and initialize a list of `Model` objects as foll
...
-        Data = new List<Sales>()
+        Data = new List<Learning>()
         {
-            new Sales(){Product = "iPad", SalesRate = 25},
-            new Sales(){Product = "iPhone", SalesRate = 35},
-            new Sales(){Product = "MacBook", SalesRate = 15},
-            new Sales(){Product = "Mac", SalesRate = 5},
-            new Sales(){Product = "Others", SalesRate = 10},
+            new Learning(){Name = "Cooking and Other Works", Value = 54},
+            new Learning(){Name = "Before Sleeping", Value = 60},
```
Comment (Saravanan-Madhesh) [inline MAUI/Pyramid-Charts/Getting-Started.md:120]: Change some other use case, we modified it as not the proper use case

File: MAUI/Pyramid-Charts/Overview.md [hunk 2]
```diff
@@ -9,12 +9,14 @@ documentation: ug
...
 
 ## Key features
 
-* User interaction - The pyramid chart selection and tooltips user interaction features significantly improves the end-user experience.
+* It supports pyramid chart for more user-friendly data representation and greater UI visualization.
 
-* Legends - Display additional information about the data points of chart segments.
+* The end-user experience is greatly enhanced by including the user interaction features such as selection and tooltip.
+
+* Legends provide more information about the segments. If an item exceeds the available bounds, the legends can be wrapped or scrolled.
```
Comment (Saravanan-Madhesh) [inline MAUI/Pyramid-Charts/Overview.md:20]: no wrap support, it only scrolled.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Pyramid-Charts/Getting-Started.md [hunk 3]
```diff
@@ -161,15 +162,15 @@ chart.BindingContext = viewModel;
 
 ## Populate chart with data
 
- Binding `Data` to the Pyramid chart ItemsSource property from its BindingContext to create our own Product Sales pyramid chart.
+ Binding `Data` to the Pyramid chart [ItemsSource]() property from its BindingContext to create our own pyramid chart.
```
[the remark this answers is not in the document]
Comment (SaiyathAliFathima) [reply MAUI/Pyramid-Charts/Getting-Started.md:165]: I have updated this change

**Review by Saravanan-Madhesh — APPROVED** [review]
The change are fine, please check in staging and update.

**Pull request merged**